lua/lspconfig/denols.lua

```diff
@@ -26,4 +81,6 @@ Deno's built-in language server
   };
 }
 
+configs.denols.buf_cache = buf_cache
```

Comment:
What is the purpose of these two lines?

Reply:
I followed the method of texlab.

https://github.com/neovim/nvim-lspconfig/blob/8f642586e4213d257099704c3dacceb7f4d64627/lua/lspconfig/texlab.lua#L107-L108

Reply:
This actually predates my maintainership so I don't know how we use these 😆 I can take a look

Reply:
I think it's fine